coldfront/core/project/templates/project/project_allocation_addition/request_card.html

```diff
@@ -52,31 +52,33 @@ <h2>
         </a>
       {% endif %}
     </p>
-    <p class="card-text text-justify">
-      <strong>Unsigned Memorandum of Understanding: </strong>
-      <a href="{% url 'service-units-purchase-request-download-unsigned-mou' addition_request.pk 'service-units-purchase' %}" class="btn btn-primary">
-        <i class="fas fa-download" aria-hidden="true"></i>
-        Download
-      </a>
-    </p>
-    {% if can_upload_mou or mou_uploaded %}
+    {% if can_download_mou %}
```

Comment:
Can you move this block into its own template that can be included? This was, the same logic won't be duplicated three times.

The URL and name of the document would need to be passed in.

(Look for examples of `request_card.html` getting included in other templates.)